Makefile

```diff
@@ -71,9 +71,10 @@ install-generic:
 		install -m 644 -D --target-directory="$(DESTDIR)/usr/share/openqa/$${f%/*}" "$$f";\
 	done
 
-	for i in db images testresults pool/1 cache webui/cache; do \
+	for i in db images testresults pool/1 cache webui/cache backup; do \
 		mkdir -p "$(DESTDIR)"/var/lib/openqa/$$i ;\
 	done
+	ln -sfn /var/lib/openqa/backup "$(DESTDIR)"/var/lib/openqa/SQL-DUMPS
```

Comment:
Do we really need this symlink?

Reply:
If we remove this symlink we have to modify the database backup script as well.

Reply:
The default [DUMP_FOLDER](https://github.com/os-autoinst/openQA/blob/master/script/dump-db#L3) is the symlink to `/var/lib/openqa/backup`, both of which aren't created by default at present during installation.

Reply:
I think it won't break anything even if we change the script to use the `backup` directory and completely ignore `SQL-DUMPS` in the script.
 `SQL-DUMPS` will keep following what's inside `backup` as it is a symlink.